frame/clearing-house/src/lib.rs

```diff
@@ -1384,6 +1511,35 @@ pub mod pallet {
 
 	// Helper functions - validity checks
 	impl<T: Config> Pallet<T> {
+		fn is_funding_update_time(
```

Comment:
after reading the function's details it was clear what this function was for but when I saw it as `Self::is_funding_update_time` I was a little confused by what it was supposed to do.

Reply:
@Cardosaum was it the same with you? This again is just my opinion and doesn't mean it has to be changed